packages/express-did-auth/test/authentication-factory.test.ts

```diff
@@ -0,0 +1,98 @@
+import authenticationFactory from '../src/factories/authentication-factory'
+import ChallengeVerifier from '../src/classes/challenge-verifier'
+import SessionManager from '../src/classes/session-manager'
+import {
+  challengeResponseFactory, Identity, identityFactory, mockedResFactory,
+  MockedResponse, modulo0Timestamp, otherSlotTimestamp
+} from './utils'
+import MockDate from 'mockdate'
+import { ErrorCodes } from '../src/errors'
+import { TokenConfig } from '../src/types'
+
+describe('authenticationFactory', () => {
+  const mockBusinessLogicFactory = (result: boolean) => async () => result
+  const challengeSecret = 'theSecret'
+  const challengeExpirationTimeInSeconds = 60
+
+  const challengeVerifier = new ChallengeVerifier({ challengeSecret, challengeExpirationTimeInSeconds })
+  const sessionManager = new SessionManager({})
+
+  let config: TokenConfig
+  let userIdentity: Identity
+
+  beforeAll(async () => {
+    const serviceIdentity = await identityFactory()
+    config = {
+      serviceDid: serviceIdentity.did,
+      serviceSigner: serviceIdentity.signer,
+      serviceUrl: 'https://the.service.com'
+    }
+
+    userIdentity = await identityFactory()
+  })
+
+  afterEach(() => MockDate.reset())
+
+  it('should return 401 if no response', async () => {
```

Comment:
In genreal, we are using `test` in all the other modules. Maybe we should keep consistence on this